core/spoofax.eclipse/src/main/java/mb/spoofax/eclipse/editor/SpoofaxEditorBase.java

```diff
@@ -159,7 +161,7 @@ public void reconfigure() {
         final Display display = Display.getDefault();
         display.asyncExec(() -> {
             sourceViewerExt2.unconfigure();
-            setSourceViewerConfiguration(new SpoofaxSourceViewerConfiguration());
+            setSourceViewerConfiguration(new SpoofaxSourceViewerConfiguration(this));
```

Comment:
I think it would be good to generate a `SpoofaxSourceViewerConfiguration` implementation for each Eclipse language plugin, allowing it to be customized, akin to `<Language>Editor` and the other generated Eclipse plugin classes. The language can then pass its own component and pie component to the implementation. `SpoofaxSourceViewerConfiguration` should do as much as possible, such that the generated class is as small as possible (preferably just a constructor that passes the component and pie component)

Reply:
The `<Language>Editor` class may then have to pass in a `Supplier<SpoofaxSourceViewerConfiguration>` which is used in place of `new SpoofaxSourceViewerConfiguration(this)`